src/Shared/Traits.cs

```diff
@@ -115,6 +115,11 @@ internal class EscapeHatches
         /// </summary>
         public readonly bool AlwaysUseContentTimestamp = Environment.GetEnvironmentVariable("MSBUILDALWAYSCHECKCONTENTTIMESTAMP") == "1";
 
+        /// <summary>
+        /// Stop truncating task inputs when logging them.
+        /// </summary>
+        public readonly bool LogTaskInputsVerbose = Environment.GetEnvironmentVariable("MSBUILDLOGTASKINPUTSVERBOSE") == "1";
```

Comment:
```suggestion
        public readonly bool LogTaskInputsVerbose = Environment.GetEnvironmentVariable("MSBUILDLOGTASKINPUTSCOMPLETE") == "1";
```

?

Reply:
Other options. Not sure I have a favorite.

MSBUILDLIMITLOGTASKINPUTS
MSBUILDLOGALLTASKINPUTS
MSBUILDLOGTASKINPUTSNOLIMIT

Reply:
I like `MSBUILDLOGALLTASKINPUTS` best I think.